examples/adminapi.py

```diff
@@ -419,17 +420,92 @@ def example_list(a, args):
         print(" {} consumer groups".format(len(groups)))
         for g in groups:
             if g.error is not None:
-                errstr = ": {}".format(t.error)
+                errstr = ": {}".format(g.error)
             else:
                 errstr = ""
 
             print(" \"{}\" with {} member(s), protocol: {}, protocol_type: {}{}".format(
-                  g, len(g.members), g.protocol, g.protocol_type, errstr))
+                g, len(g.members), g.protocol, g.protocol_type, errstr))
 
             for m in g.members:
                 print("id {} client_id: {} client_host: {}".format(m.id, m.client_id, m.client_host))
 
 
+def example_delete_consumer_groups(a, args):
+    groups = a.delete_consumer_groups(args, timeout=10)
+    for group_id, future in groups.items():
+        try:
+            response = future.result()
+            print("Deleted group with id '" + group_id + "' succesfully")
+
+        except KafkaException as e:
+            print("Failed to delete group '{}': {}".format(group_id, e))
+        except Exception:
+            raise
+
+
+def example_list_consumer_group_offsets(a, args):
+    """ List consumer group offsets
+    """
+
+    topic_partition_list = []
+    for topic, partition in zip(args[1::2], args[2::2]):
+        topic_partition_list.append(TopicPartition(topic, int(partition)))
+    if len(topic_partition_list) == 0:
+        topic_partition_list = None
+    groups = [ListConsumerGroupOffsetsRequest(args[0], topic_partition_list)]
+
+    futureMap = a.list_consumer_group_offsets(groups)
```

Comment:
this doesn't match the java api.

after looking at this for a bit, i'm suspecting the underlying librdkafka API is going to change, so i'm going to leave the review for the moment.